src/Stratis.Bitcoin.Features.SmartContracts/Models/LocalCallContractRequest.cs

```diff
@@ -5,26 +5,64 @@
 
 namespace Stratis.Bitcoin.Features.SmartContracts.Models
 {
-    public class LocalCallContractRequest
+    /// <summary>
+    /// A class containing the necessary parameters to perform a local smart contract method call request.
+    /// </summary>
+    public class LocalCallContractRequest 
     {
+        /// <summary>
+        /// The address of the smart contract containing the method.
+        /// </summary>
         [Required(ErrorMessage = "A destination address is required.")]
         [IsBitcoinAddress]
         public string ContractAddress { get; set; }
 
+        /// <summary>
+        /// The name of the method to call.
+        /// </summary>
         [Required(ErrorMessage = "A method name is required.")]
         public string MethodName { get; set; }
 
+        /// <summary>
+        /// The amount of STRAT (or sidechain coin) to send to the smart contract address. 
+        /// No funds are actually sent, but the Amount field allows
+        /// certain scenarios, where the funds sent dictates the result, to be checked.
+        /// </summary>
         public string Amount { get; set; }
 
+        /// <summary>
+        /// The gas price in Satoshi to use. This is used to calculate the expected expenditure
+        /// if the method is run by a miner mining a call transaction rather than
+        /// locally.  
+        /// </summary>
         [Range(SmartContractFormatLogic.GasPriceMinimum, SmartContractFormatLogic.GasPriceMaximum)]
         public ulong GasPrice { get; set; }
 
+        /// <summary>
+        /// The maximum amount of gas (in Satoshi) that can be spent executing this transaction.
```

Comment:
What's the problem with mentioning Satoshi? I only say this as I got confused here and was entering STRAT values when I started... Trying to avoid confusion in the reader.

Reply:
It's the wrong unit.

Reply:
Ah, got it - will remove.